[2025-04-09T06:58:35.495+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline_modo_prueba.extract_spotify manual__2025-04-09T06:58:30.412872+00:00 [queued]>
[2025-04-09T06:58:35.507+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline_modo_prueba.extract_spotify manual__2025-04-09T06:58:30.412872+00:00 [queued]>
[2025-04-09T06:58:35.508+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-04-09T06:58:35.654+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): extract_spotify> on 2025-04-09 06:58:30.412872+00:00
[2025-04-09T06:58:35.666+0000] {standard_task_runner.py:60} INFO - Started process 155 to run task
[2025-04-09T06:58:35.673+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline_modo_prueba', 'extract_spotify', 'manual__2025-04-09T06:58:30.412872+00:00', '--job-id', '68', '--raw', '--subdir', 'DAGS_FOLDER/etl_workflow.py', '--cfg-path', '/tmp/tmps75om25v']
[2025-04-09T06:58:35.676+0000] {standard_task_runner.py:88} INFO - Job 68: Subtask extract_spotify
[2025-04-09T06:58:35.788+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_pipeline_modo_prueba.extract_spotify manual__2025-04-09T06:58:30.412872+00:00 [running]> on host fc23817944d5
[2025-04-09T06:58:36.038+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='simon' AIRFLOW_CTX_DAG_ID='etl_pipeline_modo_prueba' AIRFLOW_CTX_TASK_ID='extract_spotify' AIRFLOW_CTX_EXECUTION_DATE='2025-04-09T06:58:30.412872+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-09T06:58:30.412872+00:00'
[2025-04-09T06:58:36.040+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-04-09T06:58:36.042+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /opt/***/scripts/extract_spotify.py']
[2025-04-09T06:58:36.059+0000] {subprocess.py:86} INFO - Output:
[2025-04-09T06:58:36.796+0000] {subprocess.py:93} INFO - error al extraer Spotify: [Errno 2] No such file or directory: '../data/raw/spotify/spotify.csv'
[2025-04-09T06:58:36.907+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-04-09T06:58:37.053+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=etl_pipeline_modo_prueba, task_id=extract_spotify, execution_date=20250409T065830, start_date=20250409T065835, end_date=20250409T065837
[2025-04-09T06:58:37.248+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-04-09T06:58:37.285+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
